ci/Jenkinsfile.android

```diff
@@ -54,7 +54,10 @@ pipeline {
     }
     stage('Lint') {
       steps {
-        script { cmn.nix.shell('lein cljfmt check') }
+        script {
+        cmn.nix.shell('rm -rf /home/jenkins/.m2/repository/org/clojure/clojurescript/1.10.520')
```

Comment:
What is this supposed to do?
If we need this before certain `lein` calls why not add that somewhere in `lein` task configuration? Or maybe somewhere in Nix?

Reply:
I will remove this before merging; previously I used a hack in order to force installing of `clojure/clojurescript/1.10.520` from fork, and that's way that dir contained not exactly `org.clojure/clojurescript:1.10.520`. Now it can be removed, but I'll keep it for several more runs.

Reply:
I see, thanks for the explanation. Amazing work!